### PR TITLE
fix: Fixed namespace packages + updated yarn.lock.

### PR DESCRIPTION
**Pull request by michaeldowseza**
Apologies but missed testing namespace packages in my original PR.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -136,7 +136,9 @@ function packAndInstallLibrary(name, dir, targetDir) {
     }
     fs.mkdirSync(libDestDir, { recursive: true })
 
-    const regex = new RegExp(`^${name}(.*).tgz$`)
+    const tmpName = name.replace(/[\s\/]/g, "-").replace(/@/g, "at-")
```
Comment (HaveF) [inline index.js:139]: "at-" ???  Is it right?

Comment (michaeldowseza) [reply index.js:139]: I went with restoring the original logic here. Didn't look too much into the reasoning for when you'd need to have "at". My initial testing I found like you suggested on the diff where the packed filename seems to omit it by default.

https://github.com/mweststrate/relative-deps/commit/e23ff45128346cac0d74a0f1d09c649314008f22#diff-168726dbe96b3ce427e7fedce31bb0bcL138

**Comment by chrisheninger** [comment]
Just came across this in my local development– `1.0.1` works but `1.0.2` does not work.

The error I'm getting is `TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined`

I **think** that this PR would fix my issues– but for now I've rolled back and hard locked my dependency to `1.0.1`.

**Comment by mweststrate** [comment]
Regenned the yarn file and merged. The proposed yarn file has a weird amazon proxy

**Comment by mweststrate** [comment]
Published as 1.0.3

**Comment by browniefed** [comment]
This PR broke all the things name space related.

**Comment by michaeldowseza** [comment]
Yup seems yarn and npm publish packages differently. Sigh. I wish there was a better way to test changes in this. My initial thought is to do the replacement of "@" based on if it's yarn or npm. Any thoughts or suggestions?

**Comment by browniefed** [comment]
I think that's the only option? I actually didn't realize this since I've
used yarn for so long and thus my fix for 1.0.1 sufficed for me.

**Comment by michaeldowseza** [comment]
And my experience is all in npm 😅 I'll see if I can submit a new PR tomorrow. I'm still wondering how I can properly test it all though. The last PR I basically had my project branch + 2 smaller projects where I tried to cover the basic yarn/npm cases in ubuntu and Windows but it still seemed easy to miss a very basic use case :/

**Comment by zhangwilling** [comment]
the problem is can't find  .tar. 

Use `["pack", "--filename", tmpName]` to fix it . And this way is former handler.

**Pull request merged**